### DISCUSSION TITLE
How to Enable text highlight in MDX file

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [LazyVim/LazyVim#1058] [Q&A]
I try several methods, but can't work for me. I also install remark-language-server with the Mason Install command. But I can not find the way how to config to remark-language-server and add [remark-mdx](https://github.com/mdx-js/mdx/tree/main/packages/remark-mdx) plugin.

**Comment by try-to-fly** [comment]
You can highlight MDX files in NeoVim by creating a new file `markdown.lua` in `lua/plugins`.

Here's what the file content should look like:

```lua
{
	'jxnblk/vim-mdx-js',
	{
		"instant-markdown/vim-instant-markdown",
		ft = { "markdown" },
		build = "yarn install",
		config = function()
			vim.g.instant_markdown_autostart = 0
		end,
	}
}
